test-complete/nodejs-optic-from-lexicons.js

```diff
@@ -289,17 +285,15 @@ describe('Nodejs Optic from lexicons test', function () {
           .orderBy([op.asc('popularity'), op.desc('date')])
           .select(['city', 'popularity', 'date', 'distance', 'point'])
           .prepare(0);
-        //console.log(JSON.stringify(preparedPlan));
         db.rows.query(preparedPlan, { format: 'json', structure: 'object', columnTypes: 'header' })
             .then(function (output) {
-                //console.log(JSON.stringify(output, null, 2));
                 expect(output.rows.length).to.equal(4);
                 /*expect(output[0].value['myCity.city']).to.equal('cape town');
```

Comment:
Can all this commented stuff just be deleted? It was probably commented out many years ago.